fv3core/.jenkins/actions/run_parallel_regression_tests.sh

```diff
@@ -9,12 +9,16 @@ make get_test_data
 
 if [ ${python_env} == "virtualenv" ]; then
     CONTAINER_CMD="" MPIRUN_ARGS="" make savepoint_tests_mpi
+    TARGET=init CONTAINER_CMD="" MPIRUN_ARGS="" make savepoint_tests_mpi
```

Comment:
We probably don't `get_test_data` with `TARGET=init`, so we have to make sure that is done before merging this.

Reply:
On daint you have to either download ahead of time (it is already downloaded) or run jenkins twice (which usually is the case) because each config would try to download and try and write to the same directory.

Reply:
Does that mean we need to make more changes to this PR?

Reply:
Should be good to go now.